Jenkinsfile_CNP

```diff
@@ -120,7 +120,7 @@ withPipeline("java", product, component) {
       }
       env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
       env.CCD_DATA_STORE_API_BASE_URL = "http://ccd-data-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
-      env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
+      env.MCA_API_BASE_URL = env.TEST_URL
```

Comment:
NB: When this PR runs in the pipeline this edited line will not take effect unless the number of this PR is added to: `prsToUseAat` variable on line 15.  

Would it be possible/worth assigning `env.MCA_API_BASE_URL = env.TEST_URL` in all scenarios by moving this line to 71?

Reply:
aha.. i didn't see it has multiple references. Fixed now @mattnayler